### ISSUE TITLE
Is this repo dead?

### ISSUE THREAD

**Issue by pavelbatyr** [open-mmlab/mmtracking#934]
Hello! 
Last commit in this repo was in April 23. Is mmtracking currently maintained?

**Comment by yarkable** [comment]
+1

**Comment by KunmingS** [comment]
Yes... They transfered to mmdet since mmdet3.0. No wonder so much incompatiable in this repo
